sirbot-pyslackers/endpoints/slack/messages.py

```diff
@@ -31,6 +31,88 @@ def create_endpoints(plugin):
     # stock tickers are 1-5 capital characters, with a dot allowed. To keep
     # this from triggering with random text we require a leading '$'
     plugin.on_message("s\$[A-Z\.]{1,5}", stock_quote, wait=False)
+    plugin.on_message("c\$[A-Z\.]{1,5}", crypto_quote, wait=False)
+
+
+async def crypto_quote(message, app):
+    match = STOCK_REGEX.search(message.get("text", ""))
+    if not match:
+        return
+
+    symbol = match.group("symbol")
+    LOG.debug("Fetching crypto quotes for symbol %s", symbol)
+
+    response = message.response()
+    try:
+        stocks = app["plugins"]["stocks"]
+        crypto = await stocks.crypto()
+        quote = None
+        for c in crypto:
+            if c["symbol"] == f"{symbol}USDT":
+                quote = c
+                break
+        if quote is None:
```

Comment:
Don't do it since in my opinion it make the code harder to understand but this is a good use case for a `for ... else` loop.

Reply:
Thanks for the suggestion! I will certainly keep that in mind in the future.